### ISSUE TITLE
`RandAffine`, `RandFlip`, `RandRotate` with GPU-based `torch.tensor`

### ISSUE THREAD

**Issue by wyli** [Project-MONAI/MONAI#2917]
**Is your feature request related to a problem? Please describe.**
this is as part of #2231 would potentially accelerate the current test-time augmentation pipeline, when combined with https://github.com/Project-MONAI/tutorials/blob/master/acceleration/fast_training_tutorial.ipynb

see also: #2913

cc @diazandr3s

**Comment by diazandr3s** [comment]
This is great! It will help a lot in accelerating TTA used in the MONAI Label Apps

**Comment by wyli** [comment]
addressed by #2922 #2945
